docs/reference/objects/cart/index.md

```diff
@@ -27,10 +27,10 @@ The currency code associated with the cart.
 
 ## `cart.custom_data`
 {: .d-inline-block }
-object
+array [Custom Data]({% link docs/reference/objects/cart/custom_data.md %})
```

Comment:
Can we say 'array of' please? This seems to be how we have it across these docs.